featuretools/entityset/entityset.py

```diff
@@ -1211,6 +1211,9 @@ def _import_from_dataframe(self,
                                  r.child_entity.id == entity_id]
 
         for c in dataframe.columns:
+            if not is_string(c):
+                raise ValueError("All column names must be strings. (Column has name {})".format(c))
```

Comment:
Change language to "All column names must be strings (Column {} is not a string)"